tests/catalog/test_sql.py

```diff
@@ -1158,13 +1171,13 @@ def test_list_non_existing_namespaces(catalog: SqlCatalog) -> None:
 def test_drop_namespace(catalog: SqlCatalog, table_schema_nested: Schema, table_identifier: Identifier) -> None:
     namespace = Catalog.namespace_from(table_identifier)
     catalog.create_namespace(namespace)
-    assert namespace in catalog.list_namespaces()
+    assert namespace[:1] in catalog.list_namespaces()
```

Comment:
why is this excluding the first result?

Reply:
I'm taking only the first level of the namespace.

Calling `list_namespaces()` without parameters now correctly returns only the top level namespaces.

So if `namespace` is a multi-level namespace, for example "db.ns1", only "db" is returned by `list_namespaces()`

Reply:
i see, thanks for the explanation! i think the assert here is testing that the newly created namespace exists. 
so perhaps something like this matches its behavior more

```
assert namespace in catalog.list_namespaces(namespace[:-1])
```